api/src/main/resources/application.yml

```diff
@@ -33,3 +33,5 @@ thehub:
   sync:
     domains: hub.no, thehub.dk, thehub.se, thehub.fi
     scheduling.fixedRate-in-milliseconds: 900000
+  models:
+    JobEntity.approved_at: 2000-01-01T00:00:00.000Z
```

Comment:
Note: it should look like that `job.approved-at-default: 2000-01-01T00:00:00.000Z`